config/jest.config.js

```diff
@@ -11,7 +11,7 @@ module.exports = {
         '\\.scss$': '<rootDir>/config/testMocks/styleMock.js',
         '~(.*)$': '<rootDir>/src$1'
     },
-    verbose: true,
+    verbose: false,
```

Comment:
Is this intentional? What is the motivation?

Reply:
If you want verbose testing turn it on - otherwise stop cluttering up the terminal